### DISCUSSION TITLE
Widget Themes Error

### DISCUSSION THREAD

**Discussion by jasonsan9364** [LiveHelperChat/livehelperchat#2036] [Q&A] [answered]
When I access "Widget Themes" to create a new widget, it's not loading the widget for me to customize. Asking to "Clear cookies and reload". Can you help me as I tried everything and still no success. (see the attachment)

![widget themes error](https://github.com/LiveHelperChat/livehelperchat/assets/156453430/1cb1d5e1-2d42-4be6-b07e-88de15fd144d)

**Comment by remdex** [comment]
Inspect with chrome dev toolbar or just hire developer...

**Reply by jasonsan9364** [reply]
Just checked it with the chrome dev toolbar and discussed it with the developer. However, not showing any error in Chrome. Something from backend is not running and is only shown printed code. Any help in this?

**Reply by remdex** [reply, marked as answer]
No,
There can be numerous reasons why you don't see a widget including configuration error and server env. For personal debug there is paid support. Also try to embed code directly on webpage perhps there it will work. Without direct access impossible to know.

**Reply by remdex** [reply]
Just by entering your URL in the browser I see chat started. I think you have auto chat started configuration turned on and in that scenario most likely widget theme preview won't work. NEVER touch settings you don't understand what they do.

**Reply by jasonsan9364** [reply]
How much for paid support?

**Reply by remdex** [reply]
https://livehelperchat.com/general-support-286a.html

https://livehelperchat.com/support-13c.html

But I'm off till monday :)
